### ISSUE TITLE
Installer software is not installing properly???

### ISSUE THREAD

**Issue by xpxaxsxi** [SWI-Prolog/swipl-devel#388]
I am seeing that the installer software has been changed, 

I believe I am experiencing problems, I am not totally sure, I might have multiple installations that are messing each other? Possibly I did not manually remove some previous version??? Maybe I poked things at the Registry Editor??

Here are some of the problems: pl-file extensions are not registered, user settings are not read from AppData/swipl , opening the swipl-win.exe executable starts at c:\program files\swipl\bin, prolog source files have some encoding issues...

I have been trying to install 7.7.25, as a 32-bit and 64-bit on a Windows 10 Home

**Comment by xpxaxsxi** [comment]
I installed the 7.6.4 64-bit which has the Nullsoft installer, no problems. 

Well my Windows system did get a big update yesterday, that could cause problems.

**Comment by marvin-littlewood** [comment]
Hi!

I have experienced similar when using 7.7.25-1 - 64 bit on Windows 10 Home 1803, build 17134.471.

I noticed that after I uninstalled w64-7719, the install of 7725-1 failed when trying to over-write uuid.dll. Deleting the old install folder removed this problem.

Also unlike w64-7719, the 7725-1 installer failed complaining that one of the registry keys (I think) had a filename that was too long. Originally I used: D:\Program Files\ but changing to: D:\swipl\ fixed this error.

Like @xpxaxsxi, the file extensions are not present, where they worked before.

Many thanks
